### ISSUE TITLE
Installing development version results in error for reveal

### ISSUE THREAD

**Issue by mpacer** [damianavila/RISE#280]
When running 

```
npm run reset-reveal
```

it errs 

```
> rise@0.5.0 reset-reveal /Users/mpacer/jupyter/RISE
> sed -i '11 s_^_/*_' rise/static/reveal.js/css/reveal.css

sed: -i may not be used with stdin

npm ERR! Darwin 16.5.0
npm ERR! argv "/Users/mpacer/anaconda3/envs/dev/bin/node" "/Users/mpacer/anaconda3/envs/dev/bin/npm" "run" "reset-reveal"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! rise@0.5.0 reset-reveal: `sed -i '11 s_^_/*_' rise/static/reveal.js/css/reveal.css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the rise@0.5.0 reset-reveal script 'sed -i '11 s_^_/*_' rise/static/reveal.js/css/reveal.css'.
```

**Comment by mpacer** [comment]
This is a known issue with BSD vs. GNU `sed`: 

https://stackoverflow.com/questions/21242932/sed-i-may-not-be-used-with-stdin-on-mac-os-x

The command for OSX is `sed -i '' '11 s_^_\/*_' rise/static/reveal.js/css/reveal.css`

Is there a platform agnostic version that will allow this change?

**Comment by damianavila** [comment]
Closed by #285.
